core/src/test/scala/com/rackspace/com/papi/components/checker/GivenAWadlWithMetaTransforms.scala

```diff
@@ -0,0 +1,279 @@
+//        // THEN access should be allowed;
+//        // ELSE IF the target is the overloaded target
+//        // AND the list of roles contains the overloaded role,
+//        // THEN access should be allowed;
+//        // ELSE access should be forbidden.
+//        if (targetInRoles(target, roles, List("admin", "superAdmin"))) {
+//          it should behave like accessIsAllowedWithContent(validator, method, s"/custom/metadata/${target}foo", roles, putContent, description)
+//        } else if (target.equals("service:") && targetInRoles("another_role", roles, List.empty)) {
+//          it should behave like accessIsAllowedWithContent(validator, method, s"/custom/metadata/${target}foo", roles, putContent, description)
+//        } else {
+//          it should behave like accessIsForbidden(validator, method, s"/custom/metadata/${target}foo", roles, description)
+//        }
+//      }
+//      it should behave like accessIsForbiddenWhenNoXRoles(validator, method, s"/custom/metadata/${target}foo", description)
+//    }
+//  }
+
+  targetsCustom.foreach { target =>
+    rolesList.foreach { roles =>
+      ////////////////////////////////////////
+      // IF the list of roles contains the target
+      // OR the list of roles contains admin OR superAdmin,
+      // THEN access should be allowed;
+      // ELSE IF the target is the overloaded target
+      // AND the list of roles contains the overloaded role,
+      // THEN access should be allowed;
+      // ELSE access should be forbidden.
+      ////////////////////////////////////////
+      // @TODO: Huh??? Why doesn't this work?
+      //if (targetInRoles(target, roles, List("admin", "superAdmin"))) {
+      //  it should behave like accessIsAllowed(validator, methodPut, s"/custom/metadata/${target}foo", roles, description)
+      //} else if (target.equals("service:") && targetInRoles("another_role", roles, List.empty)) {
+      //  it should behave like accessIsAllowed(validator, methodPut, s"/custom/metadata/${target}foo", roles, putContent, description)
+      ////////////////////////////////////////
+      // IF the list of roles contains admin OR superAdmin,
+      // THEN access should be allowed;
+      // ELSE access should be forbidden.
+      if (targetInRoles("", roles, List("admin", "superAdmin"))) {
+        // @TODO: Huh???
+        // @TODO: What should the content be?
+        // @TODO: Do we need the new accessIsAllowedWithContent() method?
+        // @TODO: Should we just keep the new badRequestWithStockContent() method since we know it made it all the way through?
+        //it should behave like accessIsAllowed(validator, methodPut, s"/custom/metadata/${target}foo", roles, description)
+        //it should behave like accessIsAllowedWithContent(validator, methodPut, s"/custom/metadata/${target}foo", roles, putContent, description)
+        it should behave like badRequestWithStockContent(validator, methodPut, s"/custom/metadata/${target}foo", roles, description)
```

Comment:
When using accessIsAllowed() it returns 400 with "Bad Content: Expecting the root element to be: meta:meta".